### PR TITLE
Adapt 6623 v5 refactor open text input

### PR DESCRIPTION
**Pull request by eleanor-heath**

### CODE REVIEW

**Comment by eleanor-heath** [comment]
How the component looks with the changes:
ADAPT-6623

File: package.json [hunk 1]
```diff
@@ -1,23 +1,24 @@
 {
```
Comment (cahirodoherty-learningpool) [inline package.json:1]: This file should be deleted in favour of the already existing bower.json

**Comment by simondate** [comment]
I've had some issues getting this working in v5 of the FW.

I think you need to update the buttons in your `example.json` and `properties.schema` to include an aria label.

See the MCQ component for how this is done - https://github.com/adaptlearning/adapt-contrib-mcq/blob/master/properties.schema#L230

**Comment by cahirodoherty-learningpool** [comment]
> I've had some issues getting this working in v5 of the FW. I noticed the `example.json` hadn't been updated to the v5 format.

Thanks for checking this out Simon. I'll ask Eleanor to have another look next week

**Comment by eleanor-heath** [comment]
> I've had some issues getting this working in v5 of the FW. I noticed the `example.json` hadn't been updated to the v5 format.

Hi @simondate

I have updated the example.json in my latest commit. Please let me know if you have any further issues with getting this working.

**Pull request merged**